packages/dapp/src/components/buttons/web3Modal.tsx

```diff
@@ -1,5 +1,11 @@
+import { Button, Box, Spinner, Text } from 'grommet';
+import styled from 'styled-components';
+
 import { useAppState } from '../../store';
-import { Button } from './index';
+
+const StyledText = styled(Text)`
```

Comment:
It is better to use semantic namings. If this component represents a Button's label it is better to assign a name `ButtonLabel` or similar

Reply:
not fixed yet